### PR TITLE
Improve Dataflow Framework manual

### PR DESCRIPTION
**Pull request by mernst**

### CODE REVIEW

**Comment by mernst** [comment]
@wmdietl If you don't want to review this, I think it's OK to merge without additional review.
The CI failure is a false positive that @smillst will fix before the PR is merged.

File: dataflow/manual/content.tex [hunk 1]
```diff
@@ -1573,28 +1504,25 @@ \subsection{Answering Questions}
...
-sub-node of the node \code n under consideration.  This is not limited
-to immediate children, but the abstract value for any node contained
-in \code n can be queried.
+  These transfer methods also get access to the abstract value of any
+  sub-node of the node \code n under consideration.  This is not limited
+  to immediate children, but the abstract value for any node contained
+  in \code n can be queried.
+
+\item A transfer method for procedure entry returns the initial store, given the
+  list of arguments (as \code{LocalVariableNode}s that represent the formal
```
Comment (wmdietl) [inline dataflow/manual/content.tex:1522]: The `arguments` should be `parameters`. (I see this was already wrong before.)

File: dataflow/manual/content.tex [hunk 2]
```diff
@@ -1764,17 +1681,13 @@ \section{Example: Live Variable}
...
+\textbf{Example.} An example is shown in \autoref{fig:LiveSimple}.
 
-\livevariable{Simple}{Simple sequential program to illustrate live variable.}
+\flow{LiveSimple}{.33}{1}{Simple sequential program to illustrate live variable.  Intermediate analysis results are shown.}
 
 
 \section{Default Analysis}
 
-\begin{workinprogress}
-I feel like there is a missing cross-reference or two to this section.
```
Comment (wmdietl) [inline dataflow/manual/content.tex:1776]: Is it worth keeping this at least as comment? It doesn't look like you added a reference.

File: dataflow/manual/content.tex [hunk 3]
```diff
@@ -1519,14 +1446,18 @@ \subsection{Managing Intermediate Results of the Analysis}
 \end{verbatim}
 
 Every store is associated with a particular point in the control-flow
-graph, and all stores are managed by the framework. It maintains a
-single store for every basic block that represents the information
-available at the beginning of that block.  When dataflow information
+graph, and all stores are managed by the framework. It saves an explicit store
+for the start of each basic block (\code{Analysis.stores}).
```
[the remark this answers is not in the document]
Comment (mernst) [reply dataflow/manual/content.tex:1450]: That text was already present.  I'll delete the parenthesis.

File: dataflow/manual/content.tex (hunk 2 above)
Comment (mernst) [reply dataflow/manual/content.tex:1776]: I didn't see anywhere that a cross-reference was missing.  If we notice one in the future, we can add it.

**Pull request merged**